cmd/controller-manager/app/controller-manager.go

```diff
@@ -249,6 +249,19 @@ func setInt(target *int, defaultValue int) {
 
 func setDefaultFederationConfig(fedConfig *corev1a1.FederationConfig) {
 	spec := &fedConfig.Spec
+	// TODO(sohankunkerkar) Remove when no longer necessary.
+	// This Environment variable is a temporary addition to support Red Hat's downstream testing efforts.
+	// Its continued existence should not be relied upon.
+	const defaultScopeEnv = "DEFAULT_FEDERATION_SCOPE"
```

Comment:
Please include this inside the scope in which it is used - i.e. after the if conditional (see: limit the scope of declarations to where they are used).

Reply:
Changed as you suggested.